### DISCUSSION TITLE
Extra messages aren't showing on my phone with Extra Toasts enabled on my watch

### DISCUSSION THREAD

**Discussion by SimonPhilpott** [NightscoutFoundation/xDrip#2228] [General]
Up until recently I used to get extra messages show on my phone when I'd do things like dismiss alerts on my watch. 

I'd intercept them with Tasker and perform additional tasks. They're not showing anymore. I've checked and I still have Extra Toasts enabled I mmy watch and I can't seem to figure out what's changed other than updating Xdrip.

**Comment by Navid200** [comment]
Can you try older versions of xDrip and find the last version that behaves as you describe?

**Reply by SimonPhilpott** [reply]
Any idea what the last stable version was before the latest stable release? I'll try that one as I'm almost positive it worked on that.

**Reply by Navid200** [reply]
The previous stable version was January 31, 2022.

**Reply by SimonPhilpott** [reply]
Nope. Sadly no toast messages that far back. All notifications are enabled for xdrip and wearos. All permissions too including accessibility.

**Reply by Navid200** [reply]
Do I understand correctly that going back to previous versions of xDrip, the behavior does not change?
Therefore, something other than xDrip must have changed causing the problem?
